pipes/WDL/tasks/tasks_assembly.wdl

```diff
@@ -188,14 +195,18 @@ task scaffold {
       Int          replace_length=55
       Boolean      allow_incomplete_output = false
 
+      Int?          skani_m
```

Comment:
In a future PR we should say more about these single-character params, even if it's only to have entries in the `parameter_meta` section point to the [skani docs](https://github.com/bluenote-1577/skani/wiki/skani-advanced-usage-guide).